sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -268,6 +268,14 @@ def from_connection_string(cls, conn_str, consumer_group, **kwargs):
          Greedy strategy is used by default.
         :paramtype load_balancing_strategy: str or ~azure.eventhub.LoadBalancingStrategy
         :rtype: ~azure.eventhub.EventHubConsumerClient
+        keyword str custom_endpoint_address: The custom endpoint address to use for establishing a connection to
```

Comment:
```suggestion
        :keyword str custom_endpoint_address: The custom endpoint address to use for establishing a connection to
```

Reply:
All of the lines trailing this one (272-275) will need to be indented by a single space character. Same applies to lines 277-278.